src/utils/hapticFeedbacks.js

```diff
@@ -0,0 +1,19 @@
+/* @flow strict-local */
+
+import { Platform } from 'react-native';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+
+/**
+ * Haptic feedback will trigger if iOS >= 10 OR Device >= iPhone6s.
+ *
+ * In Android feedback trigger for all devices irrespective of system settings
+ */
+export const longPressHapticFeedback = () => {
+  const hapticTriggerType = Platform.select({
+    ios: 'selection',
```

Comment:
Just FYI, this is the default that `react-native-haptic-feedback` will do on iOS if we do `longPress` — you can see that in the [upstream code](https://github.com/junina-de/react-native-haptic-feedback/blob/8e4d598ad9be886325316b0e2140df8df624a91f/ios/RNReactNativeHapticFeedback.m#L51).

I think it's still reasonable to have this, just so if that changes, we can change this ourselves, but I don't think we really need this, strictly speaking.

Reply:
Look at the next line [upstream code](https://github.com/junina-de/react-native-haptic-feedback/blob/8e4d598ad9be886325316b0e2140df8df624a91f/ios/RNReactNativeHapticFeedback.m#L54) there  it do not have default for supportsHapticFor6SAnd6SPlus, that's why I explicitly provide the `selection` type

Reply:
Ah, cool, that makes sense. Looking at [this table](https://iosref.com/ios), it seems like the 6s/6s Plus support iOS 14, which means we still support them.

It's probably worth filing a upstream bug about that, I'll go do that now.

Reply:
Looks like this is junina-de/react-native-haptic-feedback#47